src/vs/workbench/electron-sandbox/parts/titlebar/titlebarPart.ts

```diff
@@ -177,7 +179,11 @@ export class TitlebarPart extends BrowserTitleBarPart {
 				this.nativeHostService.minimizeWindow();
 			}));
 
-			// Restore
+			this._register(addDisposableListener(minimizeIcon, EventType.MOUSE_OVER, e => {
+				setupCustomHover(this.hoverDelegate, minimizeIcon, localize('window.minimize', 'Minimize'));
```

Comment:
The setupCustomHover function is already binding to the DOM event, so you don't need the additional event listener on these. You just need to register the result of the function call.